config/sync/message.template.og_membership_subscribed_approve.yml

```diff
@@ -0,0 +1,19 @@
+langcode: en
+status: true
+dependencies: {  }
+template: og_membership_subscribed_approve
+label: 'Group membership approval with subscription'
+description: 'Group privileged user accepts a membership request that is also subscribed.'
+text:
+  -
+    value: '<p>Joinup: Your request to join the @entity:bundle @entity:title was approved</p>'
+    format: full_html
+  -
+    value: '<p>@actor:full_name has approved your request to join the "@entity:title" @entity:bundle.</p><p>You will receive weekly notifications for newly created content on this collection.<br /> To manage your notifications go to "<a href="@user:my_subscriptions">My subscriptions</a>" in the user menu.</p><p>If you think this action is not clear or not due, please contact Joinup Support at <a href="@site:contact_url">@site:contact_url</a>.</p>'
```

Comment:
According to the Jira ticket this should be "... has approved your request to join **and subscribe to** the...".